tests/evaluate/test_default_evaluator.py

```diff
@@ -3198,18 +3217,18 @@ def test_evaluate_custom_metrics_string_values():
 
 
 def validate_retriever_logged_data(logged_data):
+    print("logged_data", logged_data)
```

Comment:
nit: remove debugging prints :)

Reply:
Oh I am surprised the lint didn't complain. But good catch. :) Removed